cosmic/src/kick.f

```diff
@@ -226,12 +226,15 @@ SUBROUTINE kick(kw,m1,m1n,m2,ecc,sep,jorb,vk,snstar,
           elseif(kickflag.eq.-1)then
 * Use kick scaling from Giacobbo & Mapelli 2020, Eq. 1
              vk = vk * ((m1-m1n)/mean_mej) * (mean_mns/m1n)
+             vk2 = vk*vk
           elseif(kickflag.eq.-2)then
 * Use kick scaling from Giacobbo & Mapelli 2020, Eq. 2
              vk = vk * ((m1-m1n)/mean_mej)
+             vk2 = vk*vk
           elseif(kickflag.eq.-3)then
 * Use kick scaling from Bray & Eldridge 2016, Eq. 1
              vk = alphakick * ((m1-m1n)/m1n) + betakick
+             vk2 = vk*vk
```

Comment:
Just for understanding this better - what was vk2 being set to here? a bunk value like XXXXXe-213 or something else?

Reply:
confirmed with @michaelzevin  that vk2 was being set on line 210 and not reset in the new prescription.